packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentInformation.gql.js

```diff
@@ -29,8 +36,10 @@ export const GET_PAYMENT_INFORMATION = gql`
                 }
                 telephone
             }
+            ...AvailablePaymentMethodsFragment
         }
     }
+    ${AvailablePaymentMethodsFragment}
 `;
 
 /* eslint-disable graphql/required-fields */
```

Comment:
Not related to this PR, but couldn't help but notice this rule get disabled, and never re-enabled. This is probably preventing any file after this from being properly linted. Consider fixing in this scope, or please create a followup task to resolve this.

Reply:
Good catch Tommy. Havn't noticed that. Also looks like my linter is not working 😓 . Fixed it in https://github.com/magento/pwa-studio/pull/2445/commits/06608bff1cbdf6ac405673efc877af8d56813f59